platform/working-with-vsp/policies-work-norms/deployment-policy.md

```diff
@@ -58,16 +66,12 @@ When in doubt on whether an issue is _critical_ enough for out-of-band deploymen
 1. Once approved, contact the [VSP DevOps oncall](https://dsva.pagerduty.com/schedules#PGIEA8Q) through the [#oncall channel](https://dsva.slack.com/archives/C30LCU8S3) to coordinate the release.
 1. Within two business days of this incident, send a PR with a [postmortem](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/tree/master/Postmortems), including any relevant stakeholders, VSP DEPO leadership, and any VSP team members involved in resolving the incident as reviewers.
 
-## Step-by-step process for deployment
-
-1. To deploy this change to production, the release job is run with the git SHA for the commit to deploy.
-1. The auto-deploy boolean flag must be set to `FALSE` on the job.
-1. The release tag is captured and then used as input for the matching deploy job.
-1. There are "special" jobs for `vets-website` and `vets-api` which will run both the release and deploy of `master` in one operation.
+## Resolving production bugs
```

Comment:
```suggestion
## Resolving post deploy issues 
```
This is tricky. We want to resolve issues as quickly as possible but I don't think reverting is the right call for lower priority production bugs.

Reply:
I ended up changing this heading entirely after separating it to the "resolving critical issues" doc. it's not really about "production" or "post deploy" anymore.

If we want the guidance to be "keep master deployable", to me that's equivalent to "keep master free of critical issues" (where "critical issues" is defined in this doc already). so I think the guidelines on how to fix (fix in five minutes, or revert) apply whether or not the issue has reached production.